climada/hazard/centroids/centr.py

```diff
@@ -1017,6 +1017,26 @@ def apply_point(df_exp):
                                  .map_partitions(apply_point, meta=Point)
                                  .compute(scheduler=scheduler))
 
+    @staticmethod
```

Comment:
What is the motivation to make this method static and private?

Reply:
took that over from Gabriela. Happy to remove it =)